solar/array.py

```diff
@@ -0,0 +1,71 @@
+from math import cos, sin, pi, acos, tan, asin
+from solar import SolarDay
+from cell import SolarCell
+from numpy import linspace
+from openpyxl import Workbook
+from openpyxl import load_workbook
+
+from openpyxl import cell
+
+# Read Solar Cell Angles Sheet
+filename = 'MSXIV-Strategy-Cell-Angles.xlsx'
+wb = load_workbook(filename, data_only=True)
+ws = wb.active
+
+# Create new workbook for Energy Outputs
+wb1 = Workbook()
```

Comment:
Let's talk about how to do this w/o using an Excel workbook

Reply:
New push below.